### PR TITLE
[MABL-6912][MABL-5731] Bump request timeout for deployments

### PR DESCRIPTION
**Pull request by simonychoy**
* [MABL-6912](https://mabl.atlassian.net/browse/MABL-6912) Bump the API request timeout to 30 minutes
* [MABL-5731](https://mabl.atlassian.net/browse/MABL-5731) Skip logging test status if status is null

Also updated the mabl icon to the current mabl icon (taken from mabltron repo).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.mabl.bamboo</groupId>
     <artifactId>bamboo-plugin</artifactId>
-    <version>0.1.11-SNAPSHOT</version>
+    <version>0.1.11</version>
```
Comment (DonAtMabl) [inline pom.xml:8]: Did you hand edit this or use the mvn tooling? I'm not sure if hand editing breaks anything here.

File: src/main/java/com/mabl/MablConstants.java [hunk 2]
```diff
@@ -32,8 +32,8 @@ class MablConstants {
                 PLUGIN_VERSION, System.getProperty("java.version"), getBambooVersion());
     static final String MABL_REST_API_BASE_URL = "https://api.mabl.com";
     static final Duration CONNECTION_TIMEOUT = Duration.ofSeconds(10);
-    static final Duration REQUEST_TIMEOUT = Duration.ofSeconds(30);
-    static final Duration CONNECTION_SECONDS_TO_LIVE = Duration.ofSeconds(30);;
+    static final Duration REQUEST_TIMEOUT = Duration.ofMinutes(30);
```
Comment (DonAtMabl) [inline src/main/java/com/mabl/MablConstants.java:35]: maybe leave a TODO comment that we should revisit this timeout if/when we improve the performance of the API call

**Review by DonAtMabl — APPROVED** [review]
Looks good!

File: pom.xml (hunk 1 above)
Comment (simonychoy) [reply pom.xml:8]: I hand edited this. I _thought_ this was added manually, but I'll take another look.

**Review by yawningyak — APPROVED** [review]
I see that you snuck in the new logo too :) Nice work.

File: pom.xml (hunk 1 above)
Comment (simonychoy) [reply pom.xml:8]: Oh, looking at the README, I think this is from the tooling... I'll add it back

**Pull request merged**